### PR TITLE
[DOP-3395]: Update Subdomain Links on .com/docs

### PR DESCRIPTION
**Pull request by branberry**
### Ticket

[DOP-3395](https://jira.mongodb.org/browse/DOP-3395)

### Staging URL

https://docs-mongodbcom-integration.corp.mongodb.com/DOP-3395/landing/brandonly/master/

### Notes
Added new subdomain links to use .com/docs

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -135,7 +135,7 @@ Community.
    .. card::
       :headline: Collaborative Community Forums
       :cta: Join our Community
-      :url: https://community.mongodb.com/?tck=docs_landing
+      :url: https://www.mongodb.com/docs/atlas/app-services/
```
Comment (rayangler) [inline source/index.txt:138]: Should the url here be https://www.mongodb.com/community/forums/ instead of Atlas App Sevices? Not sure if the tck should also be kept with the updated url

File: source/search.txt [hunk 2]
```diff
@@ -14,7 +14,7 @@
       :url: /view-analyze
 
    .. entry:: Start with Guides
-      :url: https://docs.mongodb.com/guides
+      :url: https://www.mongodb.com/docs/guides/?_ga=2.213835408.56545600.1669650448-540014133.1667334670
```
Comment (rayangler) [inline source/search.txt:17]: We might want to update the url in the IA entry on the `source/index.txt` page as well for consistency! The link to the guides site on that page also uses `docs.mongodb.com`

File: src/html/cloud.html [hunk 3]
```diff
@@ -381,5 +381,5 @@ <h4 class="block__title">
 
 <div
   id="navbar"
-  data-navprops='{"links": [{"url": "https://docs.mongodb.com/manual/","text": "Server"},{"url": "https://docs.mongodb.com/drivers/","text": "Drivers"},{"url": "https://docs.mongodb.com/cloud/","text": "Cloud","active": true},{"url": "https://docs.mongodb.com/tools/","text": "Tools"},{"url": "https://docs.mongodb.com/guides/","text": "Guides"}]}'
+  data-navprops='{"links": [{"url": "https://docs.mongodb.com/manual/","text": "Server"},{"url": "https://www.mongodb.com/docs/drivers/","text": "Drivers"},{"url": "https://docs.mongodb.com/cloud/","text": "Cloud","active": true},{"url": "https://docs.mongodb.com/tools/","text": "Tools"},{"url": "https://docs.mongodb.com/guides/","text": "Guides"}]}'
```
Comment (rayangler) [inline src/html/cloud.html:384]: These links are part of the old navbar for legacy-style builds ([example](https://www.mongodb.com/docs/cloud/)). Not sure if these links are out of scope or not for this ticket, but it might be good to update the other `docs.mongodb.com` links as well for consistency if we plan on updating these links too.

**Review by rayangler — COMMENTED** [review]
Looks good so far! Just a few questions and suggestions for now.

Also: Is there a staging link available for these changes? Would be good to have to visually see/interact with the changes! Let me know if you need any help with this

File: source/search.txt (hunk 2 above)
Comment (schmalliso) [reply source/search.txt:17]: We should also leave out the `_ga` parameter that keeps getting annoying tacked on to URLs, please!

File: src/html/cloud.html (hunk 3 above)
Comment (schmalliso) [reply src/html/cloud.html:384]: I thought those pages were redirected to new homes! HMM. This may require a ticket to look into since we should probably not be serving the old /docs/cloud landing.

Comment (rayangler) [reply src/html/cloud.html:384]: Yeah, I was taking a look earlier and the redirects *should* be [set up](https://github.com/mongodb/docs-landing/blob/d7da5a1af3f9cc912669402fe9be61ab21e9083a/config/redirects#L5). Would it be better to just keep the links on the html pages unchanged for now?

Comment (rayangler) [reply src/html/cloud.html:384]: This is definitely not within the scope of this ticket, but https://docs.mongodb.com/cloud actually redirects successfully 😅 I'll file a ticket

File: source/index.txt (hunk 1 above)
Comment (branberry) [reply source/index.txt]: Oops! Thank you for catching that. Based off of the [Excel sheet in the ticket](https://docs.google.com/spreadsheets/d/1NFy_cq02b6tzMCZt58LZHmaNzjZca0_rFPBBfNpyQbA/edit#gid=0), it appears that the tck is no longer needed

File: src/html/announcements/login-and-access-changes.html [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
     <ul class="toc">
       <li class="toc__item">
-        <a class="toc__link" href="https://docs.mongodb.com/manual/tutorial/getting-started/">Getting Started</a>
+        <a class="toc__link" href="https://www.mongodb.com/docs/manual/tutorial/getting-started/">Getting Started</a>
```
Comment (rayangler) [inline src/html/announcements/login-and-access-changes.html:9]: Since these links are part of old html pages we plan to redirect, it might be okay to revert these changes to links in `src/html` files.

It might also be less confusing as to why some links in these files are `mongodb.com/docs` and some are `docs.mongodb.com` if we don't include these changes altogether.

**Review by rayangler — COMMENTED** [review]
`index` and `search` pages lgtm!

I think it might make sense to not include the link changes in `src/html` files altogether since they're not the main focus of this ticket, and because it'll result in some links being `mongodb.com/docs` but others remaining as `docs.mongodb.com`. What do you think?

File: src/html/announcements/login-and-access-changes.html (hunk 4 above)
Comment (branberry) [reply src/html/announcements/login-and-access-changes.html:9]: Oops! I missed these. Will update super quick

**Review by rayangler — APPROVED** [review]
LGTM! 

Seems like there are some leftover changes from the `static` folder. They were probably changed along with the `src/html` files?
